CHANGELOG.md

```diff
@@ -23,12 +23,14 @@ package to catch or to otherwise handle errors that you may encounter.
 - **CUMULUS-2478**
   - Implemented [bucket map](https://github.com/asfadmin/thin-egress-app#bucket-mapping).
   - Implemented /locate endpoint
-  - Cumulus distribution api checks the file request against bucket map:
+  - Cumulus distribution API checks the file request against bucket map:
     - retrieves the bucket and key from file path
     - determines if the file request is public based on the bucket map rather than the bucket type
     - (EDL only) restricts download from PRIVATE_BUCKETS to users who belong to certain EDL User Groups
     - bucket prefix and object prefix are supported
   - Add 'Bearer token' support as an authorization method
+- **CUMULUS-2487**
```

Comment:
This and the associated integration test look like they were brought in from https://github.com/nasa/cumulus/pull/2306 ?

Reply:
From slack sounds like so.  I'm ignoring the spec for now.